deploy/test-environments/fleet_api/src/api/common_api.py

```diff
@@ -142,3 +142,111 @@ def get_build_info(version: str, is_snapshot: bool) -> str:
             f"API call failed, status code {api_ex.status_code}. Response: {api_ex.response_text}",
         )
         return ""
+
+
+def get_stack_latest_version():
```

Comment:
please update the return type in the signature

Reply:
updated